spec/support/bunnymock_recent_history_exchange.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+require 'bunny-mock'
+
+module BunnyMock
+  class Exchange
+    # Monkeypatch, XRecentHistory be found by the ex declare "x-recent-history"
+    def self.declare(channel, name = '', opts = {})
```

Comment:
You could write `def self.declare(channel, name = '', **opts)` which keeps the interface and is okay with rubocop.